### PR TITLE
Portable infobox inner xml

### PR DESCRIPTION
**Pull request by jacekjursza**
Fix for bug found in DAT-2745

### CODE REVIEW

File: extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php [hunk 1]
```diff
@@ -50,6 +50,14 @@ public function isEmpty( $data ) {
 		return !( isset( $data[ 'value' ] ) ) || empty( $data[ 'value' ] );
 	}
 
+	protected function getInnerXML( \SimpleXMLElement $node ) {
```
Comment (idradm) [inline extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php]: looks awesome comparing to regex, have you looked at performance?

**Comment by idradm** [comment]
LG, however I'm not sure if those are all nodes. What about default tag?

**Comment by jacekjursza** [comment]
@idradm Default tag is already handled here: https://github.com/Wikia/app/pull/7162/files#diff-b09ea7c9a303b75f3ed56e8a326e0e0cR74

File: extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php (hunk 1 above)
Comment (jacekjursza) [inline extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php]: No.  It shouldn't be relevant in such small XML as infobox template.

**Comment by idradm** [comment]
mkey

**Pull request merged**